src/components/Items/Items.tsx

```diff
@@ -0,0 +1,27 @@
+type Props = {
+  view: () => string[];
+};
+
+export const Items: React.FC<Props> = ({
+  view,
+}) => {
+  const items = view();
+
+  return (
+    <ul>
```

Comment:
There is no check if the items array is not empty before rendering the list. If the items array is empty, an empty <ul> tag will be rendered. Consider adding a condition to only render the list if the items array is not empty.